pkg/cri/config/config_unix.go

```diff
@@ -104,5 +104,7 @@ func DefaultConfig() PluginConfig {
 		ImageDecryption: ImageDecryption{
 			KeyModel: KeyModelNode,
 		},
+		EnableCDI:   false,
+		CDISpecDirs: []string{"/etc/cdi", "/var/run/cdi"},
```

Comment:
Should the default here not be `nil` allowing the defaults from the CDI package to be used? Alternatively can we use the defaults from here https://github.com/container-orchestrated-devices/container-device-interface/blob/master/pkg/cdi/spec-dirs.go#L40 directly?

Reply:
@elezar Thank you for the review! Yeah, I thought about using cdi.DefaultSpecDirs as a default and decided not to do it for better readability and consistency wit the rest of defaults. They're all clearly specified constants as far as I could see.

Reply:
Yes, it does make it clearer to just explicitly list them here.